doc/installing.rst

```diff
@@ -41,7 +41,7 @@ For accelerating xarray
 For parallel computing
 ~~~~~~~~~~~~~~~~~~~~~~
 
-- `dask.array <http://dask.pydata.org>`__ (0.9.0 or later): required for
+- `dask.array <http://dask.pydata.org>`__ (0.XX or later): required for
```

Comment:
was this intentional?

Reply:
It is the subject of #2203. I just wanted to make sure it was flagged for an update as part of this PR.